### DISCUSSION TITLE
Customizable AAD sign-in button - AAD logo

### DISCUSSION THREAD

**Discussion by ZarTrox** [Azure/api-management-developer-portal#2604] [Feature request]
## Bug description

The current AAD sign-in button contains the fixed AAD logo in front of the label.
![image](https://user-images.githubusercontent.com/58420097/220868208-d2df7b6c-87fc-46dc-8907-31881d8bb546.png)

This logo is interfering with our companies branding and styling and stands out compared to the other colors used.
We would like to further customize the AAD button, by being able to:
1. remove the AAD logo, or
2. replace the AAD logo with an own image

## Expected behavior

To disable the AAD logo a checkbox in the editor context menu for the sign-in button could be added.
Additionally, below a field for selecting the logo can be added to replace it when wanted.

## Is your portal managed or self-hosted?

Managed

**Comment by ghost** [comment]
@ZarTrox, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@ZarTrox, thank you for requesting this feature.
